prime-router/src/test/kotlin/FakeReportTests.kt

```diff
@@ -165,6 +199,72 @@ class FakeReportTests {
         assertThat(setOfStates.count() == 1).isTrue()
     }
 
+    @Test
+    fun `testBuildOrderFacilityNameAndSiteOfCareColumnIncludeNCESFacilityFlag`() {
```

Comment:
this only tests the happy path. Need to add some additional rowContexts with incorrect data(ex non-existent state or county)

Reply:
Done.